android/build.gradle

```diff
@@ -9,12 +9,36 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.3.2'
+        classpath 'com.android.tools.build:gradle:4.1.0-rc03'
```

Comment:
Can we not depend on a release candidate version? Makes me a bit uncomfortable...